cmake/options.cmake

```diff
@@ -34,10 +34,12 @@ endif()
 
 option(BUILD_TESTS          "Build tests"                                                   OFF)
 option(ENABLE_CHECK_INTERNAL "Enable internal checks"                                       OFF)
+option(ENABLE_OSS_FUZZ      "Enable the OSS-Fuzz related targets"                           ON)
```

Comment:
Does this mean that oss fuzz targets are built by default? I assume those will not be used by most devs and users.

Reply:
This doesn't change the current behavior. Just makes it possible to disable the targets so the sources do not show up twice in the `compile_commands.json` which I use for profiling.